src/main/java/seedu/address/model/profile/Email.java

```diff
@@ -15,21 +15,23 @@ public class Email {
             + "1. The local-part should only contain alphanumeric characters and these special characters, excluding "
             + "the parentheses, (" + SPECIAL_CHARACTERS + "). The local-part may not start or end with any special "
             + "characters.\n"
-            + "2. This is followed by a '@' and then a domain name. The domain name must be a valid NUS email domain "
-            + "such as:\n"
+            + "2. This is followed by a '@' and then a domain name. The domain name must one of the following "
+            + "NUS email domain:\n"
```

Comment:
Minor grammar issue.
```suggestion
            + "NUS email domains:\n"
```